Documentation for Event distribution graph (#2964)
harness/developer-hub: docs/continuous-error-tracking/getting-started/cet-event-dashboard.md

    ![Event Dsitribution Default](./static/cet-event-distribution-default.png)

### Adding an Event to the Event Graph:
  To add new event to the graph, select the little + icon from the last column as shown in the image below. Whenever you add events from the event list they will be automatically visualized in the chart. Comparing both series enables you to see the impact of the selected errors on the total.
    ![Add Exception To Graph](./static/cet-event-distribution-add-exception.png)
    ![Exception Added To Graph](./static/cet-event-distribution-exception-plotted.png)

### Plot Top Events on the Graph:
  To plot the top events on the graph you can select the **Event Distribution** split series icon. This will plot the top contributors as series on the graph. This enables you immediately see which ones have the most impact and may need to be dealt with first.
    ![Plot Top Events](./static/cet-event-distribution-plot-top-events.png)

### Removing Event from the Event Graph:
  

>   * **Remove a Single Event**:  You can remove a single event from the Event Graph by clicking again on the same icon.

Reviewer: ```suggestion
  * **Remove an event**:  To remove an event from the Event Graph select the ** - ** icon next to it.
```

    ![Remove Evenet from Graph](./static/cet-even-distribution-remove-event.png)

  * **Remove All Events**: You can remove all the event from the graph by selecting **Clear**
    ![Clear All Events](./static/cet-event-distribution-clear-events.png)

> ### Selecting Graph Timeframes:

Reviewer: ```suggestion
### Select graph timeframes
```

  You can further customize the graph by changing the timeframes by following the two methods described below:

>   * **Time period:**  Choose a period for which you want to see the data. The default option is **Last 1 hour**.

Reviewer: ```suggestion
  * **Time period**:  Select this option and choose a time period for which you want to see the data. The value is **Last 1 hour**.
```

  * **Time Slice:** Click on the graph and a small time slice pane will open. You can expand that time slice or shrink it to fit your required time frame.
    ![ZoomIn Time Slice](./static/cet-event-distribution-select-time-slice.png)

### Collapsing the Event Graph:

>   The Event Graph also lets you collapse it so that it's not displayed on the screen, and then  it when needed as shown in the image below

Reviewer: ```suggestion
You can collapse the Event Graph when you don't need.
```
